saas/api/roles.py

```diff
@@ -303,7 +303,14 @@ class AccessibleByQuerysetMixin(UserMixin):
     role_model = get_role_model()
 
     def get_queryset(self):
-        return self.role_model.objects.filter(user=self.user)
+        queryset = self.role_model.objects.filter(user=self.user)
+
+        include_personal_profile = self.request.GET.get('include_personal_profile', 'false').lower() == 'true'
```

Comment:
use `self.request.query_params` instead of `self.request.GET` per Django Rest Framework recommendations.

It might also be worth defining the API as "`include_personal_profile` evaluates to `True`". This way both `?include_personal_profile=true` and `?include_personal_profile=1` work.